py4DSTEM/process/diskdetection/diskdetection_cuda.py

```diff
@@ -326,13 +327,6 @@ def _find_Bragg_disks_single_DP_FK_CUDA(
     Returns:
         peaks                (PointList) the Bragg peak positions and correlation intensities
     """
-    assert subpixel in [
-        "none",
-        "poly",
-        "multicorr",
-    ], "Unrecognized subpixel option {}, subpixel must be 'none', 'poly', or 'multicorr'".format(
-        subpixel
-    )
```

Comment:
Is this just a general purge of assert statements from py4DSTEM?

Reply:
This only affects the CUDA disk detection, where this was getting run for every single diffraction pattern, which is a waste of time. To be safe I'll add this assert into the top level `find_Bragg_disks` function so that it always definitely gets checked once.